core/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -355,7 +355,8 @@ class Router(formatOps: FormatOps) {
           case _ => NoSplit
         }
         Seq(
-          Split(modification, 0)
+          Split(modification, 0),
+          Split(if (leftOwner.parent.exists(_.is[Mod])) Space else NoSplit, 0)
```

Comment:
Replace it with another case [here](https://github.com/scalameta/scalafmt/pull/831#discussion_r105847958). We want to change the value of `modification`.